doc/en/api/1.0.0/importer.yaml

```diff
@@ -99,14 +127,585 @@ paths:
         204:
           description: No Content.
 
+parameters:
+  AsyncParameter:
+    name: async
+    description: Run the import asyncronously. Defaults to false.
+    in: path
+    type: boolean
+    required: false
+  ExecParameter:
+    name: exec
+    description: Run the import when it is created. Defaults to false.
+    in: path
+    type: boolean
+    required: false
+  ExpandParameter:
+    name: expand
+    type: string
+    in: path
+    description: 'What level to expand the response object to. Can be "self" (expand only the response object and its immediate children), "all" (expand all children), "none" (don''t include any children), or a nonnegative integer, indicating the depth of children to expand to. Defaults to "self" or "none", depending on the request.'
```

Comment:
This doesn't work as a string enum, as it can have an infinite number of values. The rest do though.